### DISCUSSION TITLE
Get theme method?

### DISCUSSION THREAD

**Discussion by DominicCulyer** [TomSchimansky/CustomTkinter#1338] [Q&A]
Is there a method that allows you to get the current theme selected? My application lets the user change the theme, so is there a way my program can retrieve the theme chosen by the user? Doing this means I can use the same colours for matplotlib charts, which I assume don’t automatically use the colour themes selected

**Comment by abhaymakes** [comment]
You can find the current theme like this :
```python

import customtkinter as c

r = c.CTk()

theme_manager = c.ThemeManager()

print(theme_manager._currently_loaded_theme) # this will print the currently loaded theme in your app

r.mainloop()
```

**Reply by DominicCulyer** [reply]
Thanks! This works. Is there a similar method for getting the appearance mode currently selected (e.g. light_mode, dark_mode)?

**Reply by avalon60** [reply]
Dominic, try:
`mode = customtkinter.get_appearance_mode()`
